File: 코딩테스트 입문/피자 나눠 먹기(2).js
function solution(n) {
    var answer = 0;
    
    let arr = Array(n*6+1).fill(0);
    
    arr = arr.map((v,i)=>(i%6===0)?i:0).filter(v=>v!==0);            

    arr.some((e)=>{            
        if(e%n==0){            
            answer = e/6;    
            return e%n==0;
        }
    });
        
    return answer;
}

File: 코딩테스트 입문/Python/피자 나눠 먹기(2).py
def solution(n):
    answer = 1 if 6%n == 0 else lcm(6,n)//6
    return answer

def gcd(x,y):
    while(y):
        x,y=y,x%y
    return x

def lcm(x,y):
    result = (x*y)//gcd(x,y)
    return result